src/pages/cplusplus/input_and_output/index.md

```diff
@@ -4,9 +4,9 @@ title: Input and Output
 
 ## Input and Output
 
-Input stream objects in C++ use 'cin'. This is used to save input data into the program.
+Input stream objects in C++ use 'cin'. This is pronounced "see in". This is used to input user data into the program.
```

Comment:
~~input user data into the program~~ probably should be *input data into the program* as the term "user" isn't broad enough.

Reply:
Sounds good.